app/ui/src/main/java/de/mm20/launcher2/ui/launcher/search/SearchVM.kt

```diff
@@ -72,6 +75,10 @@ class SearchVM : ViewModel(), KoinComponent {
         .getHiddenItemKeys()
         .shareIn(viewModelScope, SharingStarted.Eagerly, replay = 1)
 
+    private val ranksByLaunchCount = favoritesRepository
+        .getRanksByLaunchCount()
+        .shareIn(viewModelScope, SharingStarted.Eagerly, replay = 1)
```

Comment:
Should be shared lazily, we don't need to query data when ordering by relevance is not enabled.